src/commands/inputs.rs

```diff
@@ -0,0 +1,540 @@
+//! Implementation of the `input` command.
+
+use std::collections::HashSet;
+
+use anyhow::Result;
+use anyhow::anyhow;
+use anyhow::bail;
+use clap::Parser;
+use serde_json::Map;
+use serde_json::Value;
+use serde_yaml_ng;
+use wdl::analysis::Document;
+use wdl::analysis::types::CallKind;
+use wdl::ast::AstNode;
+use wdl::ast::AstToken;
+use wdl::ast::v1::Decl;
+use wdl::ast::v1::Expr;
+use wdl::ast::v1::InputSection;
+use wdl::ast::v1::LiteralExpr;
+use wdl::ast::v1::TaskDefinition;
+use wdl::ast::v1::Type;
+use wdl::cli::Analysis;
+use wdl::cli::analysis::Source;
+
+/// Arguments for the `input` subcommand.
+#[derive(Parser, Debug)]
+pub struct Args {
+    /// A source WDL file or URL.
+    #[arg(value_name = "PATH or URL")]
+    pub source: Source,
+
+    /// The name of the task or workflow for which to generate inputs.
+    #[clap(short, long, value_name = "NAME")]
+    pub name: Option<String>,
+
+    /// Show inputs with non-literal default values.
+    #[arg(long)]
+    pub show_expressions: bool,
+
+    /// Hide inputs with default values.
+    #[arg(long, conflicts_with = "show_expressions")]
+    pub hide_defaults: bool,
+
+    /// Generate inputs for all tasks called in the workflow.  
+    #[arg(long)]
+    pub include_nested_inputs: bool,
```

Comment:
```suggestion
    pub nested_inputs: bool,
```
nit: I think this is currently too long to type and should be shortened